report.tex

```diff
@@ -281,6 +281,23 @@ \subsubsection{Deletion}
 \subsubsection{Debugging}
 The current messages to users that a scene or option is invalid only recognizes the first error. Such error mitigation indicators should find all errors before terminating to give the user a better idea of all areas in need of change.
 
+\subsection{Avatar Builder Issues}
+\subsubsection{Avatar URL}
+Manual entry for avatar URL is confusing and lacks guidance on obtaining a URL.
+
+\subsubsection{Saving the Avatar}
+The 'Next' button does not save the avatar, misleading the user.
+
+\subsubsection{Language Selector}
+Language selection issues in the avatar builder, this presents a strange bug where users are redirected unexpectedly to the outfit section of the builder when they decide to change their language. This language change also doesn't function properly, many UI text doesn't actually change to the selected language.
+
+\subsubsection{Exporting Avatars}
+Lack of export or save option for avatars. There is a need for a proper saving functionality, perhaps including unique avatar creation IDs.
+
+\subsection{Scenario Builder Issues}
+\subsubsection{Saving Projects}
+Saving a scenario shows no progress or output. There needs to be a clear saving functionality, and a proper status tracker to see progress. There is also inadequate feedback upon scenario completion.
```

Comment:
This is covered extensively in the Remidiations section, so I think it can be removed